ovos_plugin_manager/templates/audio.py

```diff
@@ -58,52 +63,143 @@ def runtime_requirements(self):
                                    no_internet_fallback=True,
                                    no_network_fallback=True)
 
-    # methods below deprecated and handled by OCP directly
-    # playlists are no longer managed plugin side
-    # this is just a compat layer forwarding commands to OCP
+    @property
+    def playback_time(self):
+        return 0
+
+    @abstractmethod
+    def supported_uris(self):
+        """List of supported uri types.
+
+        Returns:
+            list: Supported uri's
+        """
+
+    @abstractmethod
     def clear_list(self):
         """Clear playlist."""
-        msg = Message('ovos.common_play.playlist.clear')
-        self.bus.emit(msg)
 
+    @abstractmethod
     def add_list(self, tracks):
         """Add tracks to backend's playlist.
 
         Arguments:
             tracks (list): list of tracks.
         """
-        tracks = tracks or []
-        if isinstance(tracks, (str, tuple)):
-            tracks = [tracks]
-        elif not isinstance(tracks, list):
-            raise ValueError
-        tracks = [self._uri2meta(t) for t in tracks]
-        msg = Message('ovos.common_play.playlist.queue',
-                      {'tracks': tracks})
-        self.bus.emit(msg)
```

Comment:
this one should also be left there with default code and not an abstract method

Reply:
reverted with try/except import